web/source/kmwnative.ts

```diff
@@ -80,86 +80,49 @@ if(!window['keyman']['initialized']) {
+     *  (c) filename only (anything else) - prepend keyboards option to URL
      *      (e.g. default keyboards option will be set by Cloud)
-     *           
-     * @param {string}  Lfilename  keyboard file name with optional prefix                     
-     */   
-    keymanweb.getKeyboardPath=function(Lfilename) {           
-      var rx=RegExp('^(([\\.]/)|([\\.][\\.]/)|(/))|(:)');   
+     *
+     * @param {string}  Lfilename  keyboard file name with optional prefix
+     */
+    keymanweb.getKeyboardPath=function(Lfilename) {
+      var rx=RegExp('^(([\\.]/)|([\\.][\\.]/)|(/))|(:)');
       return (rx.test(Lfilename) ? '' : keymanweb.options['keyboards']) + Lfilename;
     }
 
     /**
      * Align input fields (should not be needed with KMEI, KMEA), making them visible if previously hidden.
-     * 
+     *
      *  @param  {object}   eleList    A list of specific elements to align.  If nil, selects all elements.
-     * 
+     *
      **/
     keymanweb.alignInputs = function(eleList: HTMLElement[]) {
-      if(device.touchable) {
-        var domManager = keymanweb.domManager;
-        var processList: HTMLElement[] = [];
-
-        if(eleList) {
-          // Did the user specify the actual element or the touch-alias?
-          eleList.forEach(function(element: HTMLElement){
-            if(element.base) {
-              // It's a touch-alias element, which is what we wish to perform alignment on.
-              processList.push(element);
-            } else {
-              // This retrieves an element's touch-alias, should it exist.
-              let touchAlias = element['kmw_ip'] as HTMLDivElement;
-              if(touchAlias) {
-                processList.push(element['kmw_ip']);
-              }
-            }
-          });
-        } else {
-          processList = domManager.inputList;
-        }
-
-        processList.forEach(function(element: HTMLElement) {
-          if(dom.Utils.instanceof(element, "TouchAliasElement")) {
-            (element as com.keyman.dom.TouchAliasElement).updateInput();
-          }
-          element.style.visibility = 'visible';
-          if(element.base.textContent.length > 0) {
-            element.base.style.visibility = 'hidden';
-          }
-        })      
-      }
+      // no-op
     }
 
     /**
-     * Programatically hides all input fields with underlying elements.  Restore with .alignInputs.    
-     * 
+     * Programatically hides all input fields with underlying elements.  Restore with .alignInputs.
+     *
      *  @param  {boolean}   align    align and make visible, else hide
-     * 
+     *
      **/
     keymanweb.hideInputs = function() {
-      var domManager = keymanweb.domManager;
-      if(device.touchable) {
-        for(var i=0; i<domManager.inputList.length; i++) {
-          domManager.inputList[i].style.visibility='hidden';
-          domManager.inputList[i].base.style.visibility='visible';
-        }        
-      }
+      // no-op.
```

Comment:
Can we just delete this function? Or is it also public? If so, we need to do same deprecation as with `alignInputs`

Reply:
Already done in #7360.